cdec/installation-manager-resources/src/main/resources/im-scripts/install-codenvy.sh

```diff
@@ -1524,8 +1561,8 @@ doCheckAvailableResourcesOnNodes() {
                 ;;
         esac
 
-        # check on OS CentOS 7
-        if [[ "${osType}" != "CentOS" || "7.1" > "${osVersion}" ]]; then
+        # check on CentOS or RHEL OS of version >= 7.1
+        if [[ "${osType}" != "CentOS" || "${osVersion}" < "${MIN_OS_VERSION}" ]]; then
```

Comment:
> here  we don't check RHEL ? (comment is saying checking RHEL)
In that case we are checking both RHEL OS and CentOS as well, because of command: `if [[ -f /etc/redhat-release ]]`.

> should we have a function for this test to avoid duplicates ?
we could, but there is only one duplication, so I want to have the code as clear as possible.